### PR TITLE
Delete deprecated mixins

### PR DESCRIPTION
**Pull request by SergeAstapov**
Part of #116.

### CODE REVIEW

File: tests/dummy/app/templates/usage.hbs [hunk 1]
```diff
@@ -277,17 +277,23 @@ triggerOnAlphaNumeric(event) {
...
-to reopen `Ember.TextField` and `Ember.TextArea` and apply the `EKOnInsertMixin` and `EKFirstResponderOnFocusMixin`. This ensures that whenever an
-input is focused, other key responders will not fire. This applies to `input` and `textarea` helpers:
+To prevent `ember-keyboard` from responding to keystrokes while an input/textarea is focused,
+you may use the `on-key` modifier with regular `<input>` and `<textarea>` elements and invoke
+`ekEvent.stopPropagation()` and/or `ekEvent.stopImmediatePropagation()`.
+This ensures that whenever an input is focused, other key responders will not fire.
 
 ```htmlbars
-{{input}}
+<input {{on-key '/' this.onSlash}} />
```
Comment (SergeAstapov) [inline tests/dummy/app/templates/usage.hbs:286]: @lukemelia do you mind to take a look into this example? I'm not 100% sure this is the exactly right way

Comment (lukemelia) [reply tests/dummy/app/templates/usage.hbs:286]: This looks as I would expect.

**Review by lukemelia — APPROVED** [review]

**Comment by lukemelia** [comment]
@SergeAstapov what do you want to do about the Ember 3.8 failure?

**Comment by SergeAstapov** [comment]
@lukemelia I’m working on the fix. I’ll push an update once I have it

**Comment by SergeAstapov** [comment]
@lukemelia found the solution for 3.8 scenario - in the test component used `set(this, 'prop', value)` instead of `@tracked prop` decorator as decorators support was added in Ember 3.10 (unless I'm missing something.)

**Comment by lukemelia** [comment]
Thanks, @SergeAstapov. Seems OK until we drop 3.8 support.

**Pull request merged**